pensjon-brevbaker/build.gradle.kts

```diff
@@ -8,9 +8,9 @@ val jupiterVersion: String by project
 val hamkrestVersion: String by project
 val logstashVersion: String by project
 val micrometerVersion: String by project
-val apiModelVersion: String by project
 val jacksonJsr310Version: String by project
 val mockkVersion: String by project
```

Comment:
Hva er tanken din om pensjon-brevbaker modulen? Min tanke er at hele pakken "no.nav.pensjon.template" bør skilles ut i en brevbaker-* modul, kanskje til og med samme modul som DSL. For at andre skal kunne ha sine egne maler i sitt eget repo, så må de kunne rendre til LetterMarkup.

Reply:
Litt usikker på kor vi skal setje grensa/kva vi skal få inn. Eg har tenkt at dsl-modulen har akkurat det andre treng for å lage sine malar, og at det er den malen vi aksepterer inn i våre endepunkt på eit eller anna vis. Og da at det er vi som gjer om til LetterMarkup i vår app, og da at `Letter2Markup` og liknande for rendring ligg på vår side.